docs/reference/projectyml.rst

```diff
@@ -16,11 +16,7 @@ The ``project.yml`` file defines the following settings:
 
 .. autoclass:: inmanta.module.ProjectMetadata
 
-.. autoclass:: inmanta.module.ModuleRepoInfo
-    :show-inheritance:
-
-.. autoclass:: inmanta.module.ModuleRepoType
-    :show-inheritance:
```

Comment:
Should we really get rid of this? I thought we'd just deprecate them once Hugo's PR is in?

Reply:
If you look at the rendered documentation, you see that these two don't really provide any useful information to the end-user. As such, I though it was better to remove it entirely.

Reply:
I see, and the `repo` field itself still has documentation.